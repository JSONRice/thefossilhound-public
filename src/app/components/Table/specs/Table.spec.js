// import React from "react";
// import { shallow } from "enzyme";
// import Table from "./Table";
// import data from "./story_test_data.js";

// describe("Table", () => {
//   it("renders children when passed in", () => {
//     const wrapper = shallow(
//       <Table data={tableData}>
//         {({ rows, sort, loading, filter }) => {
//           return (
//             <React.Fragment>
//               <h1>Spec</h1>
//             </React.Fragment>
//           )
//         }}
//       </Table>
//     );
//     expect(wrapper.contains(<h1>Spec</h1>)).to.equal(true);
//   });
//
//   it('renders correctly', () => {
//     const wrapper = shallow(<Table />);
//     expect(wrapper).toMatchSnapshot();
//     // On the first run of this test, Jest will generate a snapshot file automatically.
//   });
// });

describe("Addition", () => {
  it("knows that 2 and 2 make 4", () => {
    expect(2 + 2).toBe(4);
  });

  // it("we have test data", () => {
  //   expect(data).not.toBe(null);
  //   expect(data).not.toBe(undefined);
  // });

  // it("renders children when passed in", () => {
  //   const wrapper = shallow(
  //     <Table data={tableData}>
  //       {({ rows, sort, loading, filter }) => {
  //         return (
  //           <div>
  //             <h1>Spec</h1>
  //           </div>
  //         )
  //       }}
  //     </Table>
  //   );
  //   expect(wrapper.contains(<h1>Spec</h1>)).to.equal(true);
  // });
});
