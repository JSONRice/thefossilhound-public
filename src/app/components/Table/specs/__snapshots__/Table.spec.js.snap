// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table renders correctly and snapshots 1`] = `
Array [
  <h1>
    Hello Spec
  </h1>,
  <article
    className="Table__TableContainer-sc-1fgjbjf-0 jFQjgj"
  >
    <div
      className="Header-sc-1ndpwj3-0 fnmvWP"
    >
      <div
        className="Cell__StyledCell-sc-18a70e-0 iADFcx"
        onClick={[Function]}
        width="2.5%"
      >
        <div
          className="Icon__StyledIcon-sc-1k2g4fb-0 cWLQeP"
          color="#0091bc"
          fontFamily="fontFamilyGlyphiconsHalflings"
          fontSize={10}
          name="triangleRight"
        />
      </div>
      <div
        className="HeaderCell__Cell-sc-1lytdth-0 idxTJJ"
        onClick={[Function]}
        width="20%"
      >
        kingdom
      </div>
      <div
        className="HeaderCell__Cell-sc-1lytdth-0 idxTJJ"
        onClick={[Function]}
        width="20%"
      >
        phylum
      </div>
      <div
        className="HeaderCell__Cell-sc-1lytdth-0 idxTJJ"
        onClick={[Function]}
        width="20%"
      >
        class
      </div>
      <div
        className="HeaderCell__Cell-sc-1lytdth-0 idxTJJ"
        onClick={[Function]}
        width="20%"
      >
        link
      </div>
    </div>
    <section
      className="Rows__RowsContainer-sc-5mr2tl-0 iDURcl"
    >
      <div
        className="NewRow__Row-l7m67a-0 dWzBaz"
      >
        <div
          className="Cell__StyledCell-sc-18a70e-0 iADFcx"
          width="2.5%"
        />
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Animalia
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Chordata
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Chondrichthyes
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          <span
            className="StyledAppLink-sc-5zv1an-0 hisFbU"
          >
            <a
              href="/collections/chondrichthyes"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              chondrichthyes
            </a>
          </span>
        </div>
      </div>
      <div
        className="NewRow__Row-l7m67a-0 dWzBaz"
      >
        <div
          className="Cell__StyledCell-sc-18a70e-0 iADFcx"
          width="2.5%"
        />
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Animalia
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Arthropoda
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Malacostraca
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          <span
            className="StyledAppLink-sc-5zv1an-0 hisFbU"
          >
            <a
              href="/collections/malacostraca"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              malacostraca
            </a>
          </span>
        </div>
      </div>
      <div
        className="NewRow__Row-l7m67a-0 dWzBaz"
      >
        <div
          className="Cell__StyledCell-sc-18a70e-0 iADFcx"
          width="2.5%"
        />
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Animalia
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Arthropoda
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          Trilobita
        </div>
        <div
          className="Cell__StyledCell-sc-18a70e-0 bAGDjV"
          width="20%"
        >
          <span
            className="StyledAppLink-sc-5zv1an-0 hisFbU"
          >
            <a
              href="/collections/trilobita"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              trilobita
            </a>
          </span>
        </div>
      </div>
    </section>
  </article>,
]
`;
